### DISCUSSION TITLE
EfficientNetB0 weights are not downloading for me in torchvision 0.16 Nightly Build

### DISCUSSION THREAD

**Discussion by dhruvvaidh** [mrdbourke/pytorch-deep-learning#503] [Q&A]
I am currently in module 8 of the course, and I am trying to load the weights of the EfficientNetB0 model, but I have been getting the following traceback. What do you think I should do to fix it?

![error-transfer-learning](https://github.com/mrdbourke/pytorch-deep-learning/assets/68579019/9b145526-6de5-4e3a-8e37-e79684ec2afc)

![error-transfer-learning-2](https://github.com/mrdbourke/pytorch-deep-learning/assets/68579019/21f3c927-aa7c-41ef-a19f-cf5a28513f34)

**Comment by mrdbourke** [comment]
Hi @dhruvvaidh ,

Good question!

I've not found this issue before. 

I'd say it could be an issue with the nightly build.

Have you tried installing another version of PyTorch?

Or even a stable version of `torchvision` (e.g. v0.15 as of June 2023)? 

This code snippet runs fine in Google Colab: 

```python
import torch 
import torchvision

weights = torchvision.models.EfficientNet_B0_Weights.DEFAULT
model = torchvision.models.efficientnet_b0(weights=weights)

print(f"torch version: {torch.__version__}")
print(f"torchvision version: {torchvision.__version__}")
print(model)
```

Output:

```
Downloading: "https://download.pytorch.org/models/efficientnet_b0_rwightman-3dd342df.pth" to /root/.cache/torch/hub/checkpoints/efficientnet_b0_rwightman-3dd342df.pth
100%|██████████| 20.5M/20.5M [00:00<00:00, 92.3MB/s]
torch version: 2.0.1+cu118
torchvision version: 0.15.2+cu118
EfficientNet(
  (features): Sequential(
    (0): Conv2dNormActivation(
      (0): Conv2d(3, 32, kernel_size=(3, 3), stride=(2, 2), padding=(1, 1), bias=False)
      (1): BatchNorm2d(32, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      (2): SiLU(inplace=True)
      
 ...
```

See the notebook: https://colab.research.google.com/drive/1yNiIxDX8O2vAfi-C53E5ESApQsW-dqfO?usp=sharing

**Reply by dhruvvaidh** [reply]
Hi, I've uninstalled the Nightly Build and tried to use the stable version.
It's working

Thank you for your help
